src/blocks/digital-wallets/index.js

```diff
@@ -7,6 +12,12 @@ import { getSquareServerData } from '../square-utils';
 
 const squareDigitalWalletsMethod = {
 	name: 'square-credit-card',
+	title: 'Square',
+	description: __(
+		'This will show users the ApplePay, GooglePay, or Stripe Link button depending on their browser and logged in status.',
```

Comment:
I think we want a space between ApplePay and GooglePay. Also, `Stripe Link` isn't a payment method that Square provides so we should remove that:

```suggestion
		'This will show users the Apple Pay and/or Google Pay buttons depending on their browser and logged in status.',
```